packages/zilliqa-js-contract/test/fixtures.ts

```diff
@@ -24,15 +27,23 @@ contract HelloWorld
 field welcome_msg : String = ""
 
 transition setHello (msg : String)
-  welcome_msg := msg;
-  msg = {_tag : "Main"; _recipient : _sender; _amount : Uint128 0; code : set_hello_code};
-  msgs = one_msg msg;
-  send msgs
+  is_owner = builtin eq owner _sender;
+  match is_owner with
+  | False =>
+    msg = {_tag : "Main"; _recipient : _sender; _amount : Uint128 0; code : not_owner_code};
+    msgs = one_msg msg;
+    send msgs
+  | True =>
+    welcome_msg := msg;
+    msg = {_tag : "Main"; _recipient : _sender; _amount : Uint128 0; code : set_hello_code};
+    msgs = one_msg msg;
+    send msgs
+  end
```

Comment:
Do we want to standardize all `helloWorld` contracts with events instead of `send msg`?

Reply:
Currently this test contract is kept in line with whatever is on `master` of https://github.com/Zilliqa/scilla.